report/src/main/webapp/app/themes/winterEdition.directive.js

```diff
@@ -47,8 +47,10 @@ define(['angularAMD'], function (angularAMD) {
 
       if (winterEditionEnabled && isChristmas) {
         _.merge($scope.theme, WINTER_THEME);
-        $scope.$on('$viewContentLoaded', function() {
-          snowFall.snow(document.querySelector('.aside'), {flakeCount: 15, maxSpeed: 5});
+
+        var removeListener = $scope.$on('$viewContentLoaded', function() {
```

Comment:
Do we need a `removeListener ` variable?

Reply:
Yes, otherwise more snow will render everytime something is loading on page till we have a blizzard and browser slows down.